dbclient/HiveClient.py

```diff
@@ -525,6 +543,7 @@ def retry_failed_metastore_export(self, cid, failed_metastore_log_path, iam_role
                             success_item = {'table': f'{db_name}.{table_name}', 'iam': iam_role}
                             sfp.write(json.dumps(success_item))
                             sfp.write('\n')
+                            checkpoint_metastore_set.write(f'{db_name}.{table_name}')
```

Comment:
Ditto.

Reply:
added flush

Reply:
This checkpoint is written but not used?